### PR TITLE
Dns info

### PR DESCRIPTION
**Pull request by bastinc**
Add DNS addresses in InfoBox

### CODE REVIEW

**Review by cassidyjames — CHANGES_REQUESTED** [review]
When I change DNS settings for a network, this doesn't appear to be reflected here; it continues to show my router's IP address, instead.

**Comment by bastinc** [comment]
@cassidyjames did you restart the network connection ? It's needed for changes to take effect and to be reflected in the info box.

Capture d’écran de 2020-09-24 18-25-20
Capture d’écran de 2020-09-24 18-25-40

File: src/Widgets/InfoBox.vala [hunk 1]
```diff
@@ -100,6 +101,13 @@ namespace Network.Widgets {
             router.selectable = true;
             router.xalign = 0;
 
+            var dns_head = new Gtk.Label (_("DNS:"));
+            dns_head.halign = Gtk.Align.END;
+
+            dns = new Gtk.Label (null);
+            dns.selectable = true;
+            dns.xalign = 0;
+
```
Comment (jeremypw) [inline src/Widgets/InfoBox.vala:110]: The preferred way of initializing objects is now:
```
var obj = new Object (<params>) {
    prop1 = val1,
    ....
    propx = valx
};
```

**Review by jeremypw — APPROVED** [review]
Minor code style issue but as the rest of the code does not comply probably can be fixed when the code is all updated.

A changed in DNS setting is not reflected in the popover until the connection is broken and re-established but that is not caused by this PR and there is no simple way of changing this behaviour afaict.

**Pull request merged**